src/libraries/System.ComponentModel.Composition/src/System/ComponentModel/Composition/MetadataViewProvider.cs

```diff
@@ -48,13 +49,13 @@ public static TMetadataView GetMetadataView<TMetadataView>(IDictionary<string, o
                     else
                     {
                         var implementationAttribute = metadataViewType.GetFirstAttribute<MetadataViewImplementationAttribute>();
+                        Debug.Assert(implementationAttribute != null);
                         proxyType = implementationAttribute.ImplementationType;
                         if (proxyType == null)
                         {
                             throw new CompositionContractMismatchException(SR.Format(
                                 SR.ContractMismatch_MetadataViewImplementationCanNotBeNull,
-                                metadataViewType.FullName,
-                                proxyType.FullName));
```

Comment:
As per above condition `proxyType == null`, it is safe to remove it as the formatting string only expecting one parameter